src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+
+        return numbers.stream()
+                .flatMap(num -> Stream.of(num.split(",")))
+                .mapToInt(Integer::valueOf)
+                .filter(n -> (n % 2 == 0))
+                .min()
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list: "
+                        + "< Here is our input 'numbers' >" + numbers));
```

Comment:
Sorry but don't understand the purpose of this placeholder
```suggestion
                        + numbers));
```